aiida/engine/processes/functions.py

```diff
@@ -119,14 +119,15 @@ def run_get_node(*args, **kwargs):
             Run the FunctionProcess with the supplied inputs in a local runner.
 
             The function will have to create a new runner for the FunctionProcess instead of using the global runner,
-            because otherwise if this workfunction were to call another one from within its scope, that would use
+            because otherwise if this process function were to call another one from within its scope, that would use
             the same runner and it would be blocking the event loop from continuing.
 
             :param args: input arguments to construct the FunctionProcess
             :param kwargs: input keyword arguments to construct the FunctionProcess
             :return: tuple of the outputs of the process and the process node pk
             :rtype: (dict, int)
             """
+            current_runner = get_manager().get_runner()
```

Comment:
maybe move this line directly before the `if not ...`

Reply:
done